packages/router/src/grpc/offscreen-client.ts

```diff
@@ -0,0 +1,71 @@
+import { WitnessAndBuildRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1alpha1/view_pb';
+import {
+  WitnessData,
+  Action,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
+import { InternalRequest, InternalResponse } from '@penumbra-zone/types/src/internal-msg/shared';
+import {
+  ActionBuildMessage,
+  ActionBuildRequest,
+  OffscreenMessage,
+} from '@penumbra-zone/types/src/internal-msg/offscreen';
+
+const OFFSCREEN_DOCUMENT_PATH = '/offscreen.html';
+
+let active = 0;
+
+const activateOffscreen = async () => {
+  // @ts-expect-error: no types available yet
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Don't think this eslint code comment is needed, just the ts-expect-error one

Reply:
removed